protostar/starknet_gateway/gateway_facade.py

```diff
@@ -281,3 +284,44 @@ def __init__(self, function_name: str):
 class ContractNotFoundException(ProtostarException):
     def __init__(self, contract_address: AddressRepresentation):
         super().__init__(f"Tried to call unknown contract:\n{contract_address}")
+
+
+class InvalidInputException(ProtostarException):
+    pass
+
+
+def validate_deploy_input(compiled_contract_json: str, inputs: List[int]) -> None:
+    def constructor_filter(element: Dict):
+        return element["type"] == "constructor"
+
+    def get_type_size(abi: List[Dict], typename: str):
```

Comment:
This function could be put as top-level one.

Reply:
done